lib/discover_reddit_bot/bot.ex

```diff
@@ -28,19 +28,21 @@ defmodule DiscoverRedditBot.Bot do
   end
 
   def handle({:text, text, _msg}, context) do
-    %{subreddits: subreddits_detected, urls: urls} = Parser.get_subreddits(text)
-    message = TextFormatter.format_subreddits(subreddits_detected, urls)
+    subreddits_detected = Parser.get_subreddits(text)
+    message = TextFormatter.format_subreddits(subreddits_detected)
 
-    if subreddits_detected != [] do
+    if subreddits_detected != %{} do
       answer(context, message, parse_mode: "Markdown")
+    else
+      answer(context, TextFormatter.no_subreddits_text(), parse_mode: "Markdown")
```

Comment:
I want to add this bot to groups, and I don't know if having the bot saying _"Yo I did not find sh*t in that"_ every time It does not find anything makes sense. What do you think @rockneurotiko ?

Reply:
Ahhh that makes sense :sisi: I didn't thought on that